lib/bootloader_setup.pm

```diff
@@ -220,7 +220,7 @@ sub pre_bootmenu_setup {
         sleep 60;
         return 3;
     }
-    if (get_var("USBBOOT")) {
+    if (isotovideo::get_version() < 12 && get_var("USBBOOT")) {
```

Comment:
Could you add a comment explaining the decision here please? I think your commit message explains it well enough